VigneshApp/src/app/cart-count/cart-count.component.html

```diff
@@ -1,7 +1,11 @@
 <div class="flex flex-col items-center">
     <p class="text-5xl">{{cartCount}}</p>
     <div class="flex my-5">
-        <button class="w-10 text-4xl pb-2 px-1 font-bold m-4 border border-solid border-red-500 rounded-xl shadow-sm shadow-red-500 hover:bg-red-500 hover:text-white" (click)="addCartCount()">+</button>
-        <button class="w-10 text-4xl pb-2 px-1 font-bold m-4 border border-solid border-red-500 rounded-xl shadow-sm shadow-red-500 hover:bg-red-500 hover:text-white" (click)="removeCartCount()">-</button>
+        <button
+            class="w-10 text-4xl pb-2 px-1 font-bold m-4 border border-solid border-red-500 rounded-xl shadow-sm shadow-red-500 hover:bg-red-500 hover:text-white disabled:bg-red-200 disabled:text-red-400"
+            [disabled]="cartCount >= 5" (click)="addCartCount()">+</button>
```

Comment:
Why ngClass and ngStyle not used?

Reply:
Previously I tried with css itself and for the last week task I used ngClass and ngStyle